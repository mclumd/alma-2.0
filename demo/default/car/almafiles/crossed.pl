obs(quote(rel(electric_car, coupe))).
obs(quote(rel(gas_tank_car, coupe))).

obs(quote(not(rel(charged, coupe)))).
obs(quote(rel(fueled, coupe))).

obs(quote(rel(electric_car, convertible))).
obs(quote(rel(gas_tank_car, convertible))).

obs(quote(rel(charged, convertible))).
obs(quote(not(rel(fueled, convertible)))).
