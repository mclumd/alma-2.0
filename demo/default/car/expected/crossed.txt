rel(car, coupe)
rel(gas_tank_car, coupe)
rel(electric_car, coupe)
~rel(charged, coupe)
rel(fueled, coupe)
rel(powered, coupe)
rel(car, convertible)
rel(gas_tank_car, convertible)
rel(electric_car, convertible)
rel(charged, convertible)
~rel(fueled, convertible)
rel(powered, convertible)
abnormal(convertible, gas_tank_car, "rel(powered, convertible)")
